scene/3d/node_3d.cpp

```diff
@@ -149,7 +149,11 @@ void Node3D::_notification(int p_what) {
 
 			if (data.top_level && !Engine::get_singleton()->is_editor_hint()) {
 				if (data.parent) {
-					data.local_transform = data.parent->get_global_transform() * get_transform();
+					if(!data.top_level){
```

Comment:
More style issues:
```suggestion
					if (!data.top_level) {
```

You can run clang-format locally to prevent these: https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#using-clang-format-locally

Reply:
Thank you for pointing that out. I'll make sure I do that first from now on. I had assumed my IDE was taking care of it, but clang-format wasn't setup properly.